airqo-platform/AirQo-api
# Chore cloud bigtable

- What does this PR do?
This PR is a chore that showcases the utilisation of the Cloud BigTable client library using nodeJS
- What JIRA task is related to this PR?
No JIRA card for this chore
- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/auth-service` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run dev` to test out the end points using any REST client tool available to you.
- Which end points should be ready for testing?
Base URL: http://localhost:3000

[POST] partial url: /api/v1/createTest
[GET] partial url: /api/v1/getOneTest
[GET] partial url: /api/v1/getAllTest
